### DISCUSSION TITLE
Make -popup_reminder respect the -position flag

### DISCUSSION THREAD

**Discussion by mm2270** [IBM/mac-ibm-notifications#78] [Ideas]
Hi, I'd like to request that in a future update, the `-popup_reminder` option respect the `-position` flag being used in the script.

If a pop up comes up and I move that pop up to the bottom right corner of my screen (or anywhere outside my normal view), when the popup reminder kicks in, the IBM Notifier window pops up in the same moved location. It doesn't respect the position flag, if for example I have it set to **center**. Having it respect this would be beneficial to "nag" users about something since no matter where on their screen they move the unattended window it will always come up dead center (or whatever position you've set it to), making it impossible to ignore.

Is it possible to add something like this in a future version?

**Comment by mm2270** [comment]
I just wanted to add that there is a current workaround to this, at least for certain types of popup's. I can use a function that loops if the response to the window is not what we expect, and just use a normal `-timeout `option instead of `-popup_reminder`. The window times out after X seconds, checks the input from the user and if blank or not the expected response, can call the function again and the popup window shows in the center of the screen again. Rinse, repeat until we get the expected response or acknowledgement.

Still, it would be nice to have an option for `-popup_reminder` respect the positioning of the windows as specified in the script. Having that would eliminate the need for the above workaround, and could be useful in more popup accessory types.

**Comment by SMartorelli** [comment]
Hi @mm2270 yeah it should be possible to replicate the behaviour you suggested and so I'm going to create a Feature Request from this discussion so that you can track when this feature will be implemented in IBM Notifier. Thanks for providing feedbacks!

**Comment by SMartorelli** [comment]
https://github.com/IBM/mac-ibm-notifications/issues/80

**Reply by mm2270** [reply]
Thanks! 👍
